### DISCUSSION TITLE
Locating Nielm's "Translate BTV XML" Program

### DISCUSSION THREAD

**Discussion by RawPhotoLab** [OpenSageTV/Community#35] [General]
Does anyone know where where I might find a copy of Nielm's "Translate BTV XML" Program? (nielm_translateBtvXml.jar)
It is mentioned [here](http://www.geektonic.com/2008/01/how-to-convert-beyond-tv-shows-into.html) but the [original link](http://forums.sagetv.com/forums/downloads.php?do=file&id=202) is dead.
Trying to import my BTV recordings to SageTV and this is the last part of the puzzle.
Thanks,
Frank

**Comment by wayner9** [comment]
Wow BTV!  I used to use that briefly back in the day, but haven't heard it mentioned in years.  As long as you have a good naming structure then BMT may be able to load metadata into SageTV.

**Reply by RawPhotoLab** [reply]
I tried BMT but it couldn't find related metadata for my old BTV recordings. I have created the BTV Negociator XML file and the data looks good. But I think I need Nielm's "Translate BTV XML" Program. My other choice is to translate the XML file myself, but I'm not finding good SageXML documentation to create an xml file the sagexmlinfo plugin can import.

I've been using BTV for ≈20 years, but BTV does not seem to like my new server hardware. STV is working so I'm just down to this minor issue.

**Reply by Zogg44** [reply]
Hi RPL,

Here is the file that you are looking for. I used it back in 2016 when I made the switch from BTV to SageTV. It worked pretty well IIRC but it's been a while. Good luck!

[nielm_translateBtvXml.zip](https://github.com/OpenSageTV/Community/files/11690265/nielm_translateBtvXml.zip)

**Reply by RawPhotoLab** [reply]
Hi Zogg! That worked perfectly. I was able to painlessly import about 95% of my BTV recordings. Thank you!

**Reply by Zogg44** [reply]
Great, glad it worked for you!
